core/frontend/src/components/vehiclesetup/configuration/failsafes/Failsafes.vue

```diff
@@ -0,0 +1,202 @@
+        },
+        {
+          name: 'Pilot Input Loss',
+          generalDescription: 'Triggers when the vehicle does not receive any pilot input for a given '
+          + 'amount of time.',
+          image: (await import('@/assets/img/configuration/failsafes/pilot-input.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer-outline',
+              name: 'FS_PILOT_TIMEOUT',
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PILOT_INPUT',
+            },
+          ],
+        },
+        {
+          name: 'Leak Detection',
+          generalDescription: 'Triggers when a leak is detected. We recommend keeping control electronics dry.',
+          image: (await import('@/assets/img/configuration/failsafes/leak.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Leak probe 1 type',
+              name: 'LEAK1_TYPE',
+            },
+            {
+              replacementTitle: 'Leak 1 pin',
+              name: 'LEAK1_PIN',
+            },
+            {
+              replacementTitle: 'Leak 1 logic level when dry',
+              name: 'LEAK1_LOGIC',
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_LEAK_ENABLE',
+            },
+          ],
+        },
+        {
+          name: 'Excess Internal Pressure',
+          generalDescription: 'Triggers when the internal pressure is too high. This may help to detect a leak, '
+          + 'and to avoid rapid unplanned disassembly.',
+          image: (await import('@/assets/img/configuration/failsafes/pressure.svg')).default as string,
+          params: [
+
+            {
+              replacementTitle: 'Maximum internal pressure',
+              icon: 'mdi-gauge-full',
+              name: 'FS_PRESS_MAX',
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PRESS_ENABLE',
+            },
+          ],
+        },
+        {
+          name: 'Low Battery',
+          generalDescription: 'Triggers when the voltage goes below specified thresholds.\n This can help to avoid '
+          + 'damage to the battery and potentially loss of the vehicle.',
+          image: (await import('@/assets/img/configuration/failsafes/battery.svg')).default as string,
+          params: [
+            // TODO: add support for the MAH params and coloumb counting
+            {
+              replacementTitle: 'Voltage measurement source',
+              icon: 'mdi-source-branch',
+              name: 'BATT_FS_VOLTSRC',
+            },
+            {
+              replacementTitle: 'Low voltage threshold',
+              icon: 'mdi-battery-30',
+              name: 'BATT_LOW_VOLT',
+            },
+            {
```

Comment:
```suggestion
            },
            {
              replacementTitle: 'Low voltage timeout',
              icon: 'mdi-timer-outline',
              name: 'BATT_LOW_TIMER',
            },
            {
```
Realised this one got missed. Not sure if that was intentional 🤷‍♂️

Reply:
I hadn't seem this one, interesting!